app/views/helpers/ErrorMapper.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import play.api.data.FormError
+
+object ErrorMapper {
+
+  def eoriOrAddressErrors(groupKey: String, errors: Seq[FormError]) =
```

Comment:
Is this `def` actually used somewhere?

Reply:
No not atm, but as this fix is to do with bringing SFUS into line with our other UI services I thought it prudent to keep these common functions available across all of our services (as our common tool kit if you like).